sound/soc/sof/ipc4-loader.c

```diff
@@ -190,6 +191,7 @@ static int sof_ipc4_query_fw_configuration(struct snd_sof_dev *sdev)
 			break;
 		case SOF_IPC4_FW_CFG_TRACE_LOG_BYTES:
 			trace_sof_ipc4_fw_config(sdev, "Trace log size", *tuple->value);
+			ipc4_data->mtrace_log_bytes = *tuple->value;
```

Comment:
@yongzhi1  this value is not correct in your test ?

Reply:
In my test, the code does not hit line 192, seems there is no SOF_IPC4_FW_CFG_TRACE_LOG_BYTES defined in FW configuration.